README.md

```diff
@@ -350,17 +350,24 @@ domain=127.0.0.1
 port=8443
 scheme=http
 secret=the_secret
+graphqlHost=127.0.0.1
+graphqlPort=443
```

Comment:
```suggestion
graphqlPort=8084
```
Just to match up with the recommended Fauna Dev configuration: https://docs.fauna.com/fauna/current/integrations/dev#ports